en/pocket_tos.md

```diff
@@ -0,0 +1,181 @@
+
+You may not use, export, re-export, import or transfer the Pocket Technologies except as authorized by United States law, the laws of the jurisdiction in which you obtained the Pocket Technologies, and any other applicable laws. In particular, but without limitation, the Pocket Technologies may not be exported or re-exported: (a) into any United States embargoed countries; or (b) to anyone on the U.S. Treasury Department’s list of Specially Designated Nationals or the U.S. Department of Commerce’s Denied Person’s List or Entity List. By using the Pocket Technologies, you represent and warrant that you are not located in any such country or on any such list. You also may not use the Pocket Technologies for any purpose prohibited by U.S. law, including the development, design, manufacture or production of missiles, nuclear, chemical or biological weapons. We reserve the right to limit, in our sole discretion, the availability of the Pocket Technologies or any portion thereof, to any person, geographic area, or jurisdiction, at any time.
+
+We reserve the right to limit, in our sole discretion, the availability of the Pocket Technologies or any portion thereof, to any person, geographic area, or jurisdiction, at any time.
+
+## U.S. GOVERNMENT RESTRICTED RIGHTS
+
+If this product is acquired under the terms of a government contract, use, duplication and disclosure are subject to the terms of this license and the following restrictions: subdivision (c)(1)(ii) of the Rights in Technical data and Computer Software clause at 252.227-7013 (DOD contracts); subdivisions (a) through (d) of 52.227-19 (Civilian agency contracts); and the applicable ADP Schedule Contract (GSA contracts).
+
+## ACCESS BY CHILDREN
+
+In the United States, if you are a device manufacturer or other third party (the "Sponsor") that provides devices to children under the age of 13 that utilize the Pocket Technologies, you assume the responsibility for complying with the U.S. Children’s Online Privacy Protection Act ("COPPA") and, to the extent applicable, The Family Educational Rights and Privacy Act ("FERPA"). This means that the Sponsor must notify parents/guardians of the information to be collected and obtain parental/guardian consent before collecting and sharing with the Pocket application the personal information of children under the age of 13 in order to establish an account or use the Pocket Technologies. Schools may under appropriate circumstances provide such consent on behalf of parents/guardians. For more information on complying with COPPA, see the Federal Trade Commission’s website at [https://www.ftc.gov/privacy/coppafaqs.shtm](https://www.ftc.gov/privacy/coppafaqs.shtm).
```

Comment:
This ftc.gov URL currently 404s. We need to find an alternative (I've asked the Pocket team if https://www.ftc.gov/business-guidance/resources/complying-coppa-frequently-asked-questions might work)